Cleanup AWS credentials for style and consistency (#1562)
fluent/fluent-bit-docs: administration/aws-credentials.md

# AWS Credentials

> Plugins that interact with AWS services fetch credentials from various providers in
> the following order. Only the first provider that's able to provide credentials will
> be used.

Reviewer: ```suggestion
Plugins that interact with AWS services fetch credentials from the following providers
in order. Only the first provider that provides credentials is used.
```

- [Environment variables](#environment-variables)
- [Shared configuration and credentials files](#shared-configuration-and-credentials-files)
- [EKS Web Identity Token (OIDC)](#eks-web-identity-token-oidc)
- [ECS HTTP credentials endpoint](#ecs-http-credentials-endpoint)
- [EC2 Instance Profile Credentials (IMDS)](#ec2-instance-profile-credentials-imds)

All AWS plugins additionally support a `role_arn` (or `AWS_ROLE_ARN`, for
[Elasticsearch](../pipeline/outputs/elasticsearch.md)) configuration parameter. If

> specified, the fetched credentials will then be used to assume the given role.

Reviewer: ```suggestion
specified, the fetched credentials are used to assume the given role.
```

## Environment variables

Plugins use the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` (and optionally
`AWS_SESSION_TOKEN`) environment variables if set.

## Shared configuration and credentials files

Plugins read the shared `config` file at `$AWS_CONFIG_FILE` (or `$HOME/.aws/config`),
and the shared credentials file at `$AWS_SHARED_CREDENTIALS_FILE` (or
`$HOME/.aws/credentials`)  to fetch the credentials for the profile named

> `$AWS_PROFILE`  or `$AWS_DEFAULT_PROFILE` (or "default"). See

Reviewer: ```suggestion
`$AWS_PROFILE` or `$AWS_DEFAULT_PROFILE` (or "default"). See
```
